jsonrpc/src/methods/beacon.json

```
@@ -99,16 +99,13 @@
   },
   {
     "name": "portal_beaconOffer",
-    "summary": "Send an OFFER request with given ContentKey, to the designated peer and wait for a response.",
+    "summary": "Send an OFFER request with given array of Content Pairs (Keys & Values), to the designated peer and wait for a response. `portal_historyOffer` should only accept 1 to 64 content pairs else the client will throw an out of bounds error.",
```

The text is quite ambiguous right now regarding the limits/failure case:
- It is not clear right now if it is allowed to fail on 2 content items (it is not, only when > 64)
- `should` is not sufficient, must be `must`
-  it must be made clear that it is the rpc client that will fail in case > 64 items, not the receiving client on the p2p network.

```suggestion
    "summary": "Send an OFFER request with given array of content items (keys & values), to the designated peer and wait for a response. The client MUST return an error if more than 64 content items are provided.",
```